cpp/src/parquet/file_deserialize_test.cc

```diff
@@ -753,7 +755,19 @@ TEST_F(TestPageSerde, Compression) {
 
     ResetStream();
   }
-}  // namespace parquet
+}
+
+TEST_F(TestPageSerde, Compression) {
+  // The pages keep getting larger
```

Comment:
Can you add a reference to the GH bug id so as to track what motivates these tests?

Reply:
Sure. Previously here is for testing crc and compression. I'll add motivation for my new tests here